File: solutions/python3/medium/write-a-function/solution-approach1.py
# ──────────────────────────────────────────────────
# Problem     Write a function
# Difficulty  Medium
# Subdomain   Introduction
# Platform    HackerRank
# Language    python3
# Status      Accepted
# Submitted   2026-06-21, 10:14 p.m.
# ──────────────────────────────────────────────────

def is_leap(year):
    leap = False
    if year % 400 == 0:
        return True
    elif year % 100 == 0:
        return False
    elif year % 4 == 0:
        return True
    else:
        return False
    
    return leap
    
def is_leap(year):
    return year % 4 == 0 and (year % 100 != 0 or year % 400 == 0)

File: solutions/java/easy/java-loops/solution-approach1.java
// ──────────────────────────────────────────────────
// Problem     Java Loops II
// Difficulty  Easy
// Subdomain   Introduction
// Platform    HackerRank
// Language    java
// Status      Accepted
// Submitted   2026-05-30, 12:01 a.m.
// ──────────────────────────────────────────────────

import java.util.*;
import java.io.*;

class Solution{
    public static void main(String []argh){
        Scanner in = new Scanner(System.in);
        int t=in.nextInt();
        int prev_sum;
        for(int i=0;i<t;i++){
            int a = in.nextInt();
            int b = in.nextInt();
            int n = in.nextInt();
            prev_sum=a;
            for(int j=0;j<n;j++){
                prev_sum+=(Math.pow(2,j))*b;
                System.out.print(prev_sum+" ");
            }
            System.out.println();
            
        }
        in.close();
    }
}
